app/Repository/Eloquent/SavedLearningItemRepository.php

```diff
@@ -4,20 +4,23 @@
 
 namespace App\Repository\Eloquent;
 use App\Services\EvidenceFileHandler;
-use App\UserSetting;
 use App\SavedLearningItem;
-use Carbon\Carbon;
-use Illuminate\Support\Facades\DB;
+use App\Services\CurrentUserResolver;
 
 class SavedLearningItemRepository
 {
+    /**
+     * @var CurrentUserResolver
+     */
+    private $currentUserResolver;
     /**
      * @var EvidenceFileHandler
      */
     private $evidenceFileHandler;
 
-    public function __construct(EvidenceFileHandler $evidenceFileHandler)
+    public function __construct(CurrentUserResolver $currentUserResolver, EvidenceFileHandler $evidenceFileHandler)
     {
+        $this->currentUserResolver = $currentUserResolver;
```

Comment:
Seems like the `currentUserResolver` is not necessary in this class, right?

Reply:
We need the student (current user) in both index() and createItem() methods: 
`$student = $this->currentUserResolver->getCurrentUser();`
I'm not sure if there is another way to get the student without using the `currentUserResolver`.